nixfmt.cabal

```diff
@@ -5,20 +5,19 @@ synopsis:            An opinionated formatter for Nix
 description:
   A formatter for Nix that ensures consistent and clear formatting by forgetting
   all existing formatting during parsing.
-homepage:            https://github.com/serokell/nixfmt
-bug-reports:         https://github.com/serokell/nixfmt/issues
+homepage:            https://github.com/NixOS/nixfmt
+bug-reports:         https://github.com/NixOS/nixfmt/issues
 license:             MPL-2.0
 license-file:        LICENSE
-author:              Lars Jellema
-maintainer:          lars.jellema@gmail.com
-copyright:           2022 Serokell, 2022 Lars Jellema
+author:              Serokell <hi@serokell.io> and nixfmt contributors
+copyright:           Serokell <hi@serokell.io> and nixfmt contributors
```

Comment:
Oh nice this is the counterpart of #183 

In this case here only @Lucus16 is explicitly listed, who already [confirmed it's okay](https://github.com/NixOS/nixfmt/issues/169#issuecomment-2008122683).